acl/src/main/java/org/apache/rocketmq/acl/plain/PlainAccessValidator.java

```diff
@@ -53,15 +53,17 @@ public AccessResource parse(RemotingCommand request, String remoteAddr) {
             accessResource.setWhiteRemoteAddress(remoteAddr);
         }
 
+        accessResource.setRequestCode(request.getCode());
+
         if (request.getExtFields() == null) {
-            throw new AclException("request's extFields value is null");
+            //If request's extFields is null,then return accessResource directly(users can use whiteAddress pattern)
+            return accessResource;
         }
```

Comment:
It is vulnerable to return fast here, for it may miss something if adding some logic after the return,  the  "accessResource.setRequestCode(request.getCode())" is the case, but not the only case.
As the following code relies on that the getExtFields is not null, then use if statement is suggested, just as:
if (request.getExtFields() != null) {
    accessResource.setAccessKey(request.getExtFields().get(SessionCredentials.ACCESS_KEY));
   XXX
}

Reply:
Okay,I will return the variable later in my next commits.

Reply:
Hi, @dongeforever I return back my pr's codes to original ones.